### ISSUE TITLE
Adding Map deck.gl

### ISSUE THREAD

**Issue by PRijnbeek** [EHDEN/cdm-bi#6]
I added a table "datasources" in my database with source, longitude, lattitude, nrpatients as a test.

I like to shop this database on the map using the deck.gl Grid

For this you need a API token from Mapbox.

I got one and added this in superset_config.py
MAPBOX_API_KEY = '  '

i then rebuilt the superset image:

docker-compose run --rm superset ./docker-init.sh
docker-compose up -d

but still get this when i try to show the map:

<img width="658" alt="image" src="https://user-images.githubusercontent.com/7252105/68993216-cce55980-0875-11ea-9e77-50b56f610c1c.png">

Any ideas?

**Comment by PRijnbeek** [comment]
was working in another VM :) let me try again

**Comment by PRijnbeek** [comment]
Fixed. Pretty cool.

You can also do full street view etc.

Probably also easy to add filter if we would like to.

We can customise the click and hover information from the database :)

<img width="1527" alt="image" src="https://user-images.githubusercontent.com/7252105/68994381-a5e15480-0882-11ea-9a1e-2df9819c0a60.png">

**Comment by joaorafaelalmeida** [comment]
Hi Peter,
I am glad that you figured out the problem. I never used this type o chart. I think the filters will work as they do in the other charts. We may only need to add the same SQL query in the filter chart type and set up for what we want to filter. So, since you fixed the problem, I will close the issue.
